default_yaml_config.yaml

```
@@ -114,4 +114,4 @@ screenshot: {
   'intensity': 'normal',
   'timeout': 10,
   'threads': 40
-}
+}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)